### PR TITLE
add OpenMandriva distro

### PR DESCRIPTION
**Pull request by AngryPenguinPL**
This adding OpenMandriva distro to Flatpak Quick Setup page.
Also fixing https://github.com/flatpak/flatpak.github.io/issues/280

### CODE REVIEW

File: data/distro.yml [hunk 1]
```diff
@@ -274,6 +274,15 @@
...
         <p>Note: graphical installation of Flatpak apps may not be possible with Mageia.</p>'
+        
+name: OpenMandriva Lx
+  logo: "openmandriva.svg"
+  introduction:
+    <h2>Flatpak support is built into OpenMandriva for all actively supported versions, starting from the stable/fixed release 'Rock 5.0,' through the development release 'Cooker,' and ending with the rolling release 'ROME'."</h2>
+    <p>"Flatpak comes with the pre-configured Flathub repository."</p>
+  steps:
+    - name: How to install Flatpak packages
+      text: "<p>Flatpak packages can be installed using the command line as well as graphical tools such as Discover for the Plasma release and GNOME Software for the GNOME release. Additionally, there are also tools available such as YumexNG installable via <terminal-command> sudo dnf install yumex </terminal-command> and our custom FlatDrake <terminal-command> sudo dnf install flatdrake </terminal-command> .</p>"
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
```
We want the quick steps to be simple and beginner friendly. If it is possible to install Flatpak using GUI, there is no need to mention it. Power users can figure out the additional tools and terminal commands themselves (or directly from the distribution documentation).

File: data/distro.yml [hunk 2]
```diff
@@ -274,6 +274,15 @@
       text: '
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
         <p>Note: graphical installation of Flatpak apps may not be possible with Mageia.</p>'
+        
+name: OpenMandriva Lx
+  logo: "openmandriva.svg"
+  introduction:
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
  introduction: >
```

File: data/distro.yml [hunk 3]
```diff
@@ -274,6 +274,15 @@
...
+    <h2>Flatpak support is built into OpenMandriva for all actively supported versions, starting from the stable/fixed release 'Rock 5.0,' through the development release 'Cooker,' and ending with the rolling release 'ROME'."</h2>
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
    <h2>Flatpak support is built into OpenMandriva for all actively supported versions, starting from the stable/fixed release 'Rock 5.0', through the development release 'Cooker', and ending with the rolling release 'ROME'.</h2>
```

File: data/distro.yml [hunk 4]
```diff
@@ -274,6 +274,15 @@
...
+    <p>"Flatpak comes with the pre-configured Flathub repository."</p>
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
    <p>Flatpak comes with the pre-configured Flathub repository.</p>
```

**Comment by AngryPenguinPL** [comment]
ok, fixed

File: data/distro.yml [hunk 5]
```diff
@@ -274,6 +274,12 @@
       text: '
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
         <p>Note: graphical installation of Flatpak apps may not be possible with Mageia.</p>'
+        
+name: OpenMandriva Lx
```
Comment (AsciiWolf) [inline data/distro.yml]:
```suggestion
- name: OpenMandriva Lx
```

One more typo.

**Comment by AsciiWolf** [comment]
Looks good now, thanks!

**Comment by AsciiWolf** [comment]
Although I am not sure about the (missing) svg logo border. Anyway, let's merge this and we'll see how it will look.

**Pull request merged**